tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import { getBlockPagePermalink } from '../../../utils';
+
+const block = {
+	name: 'Mini Cart Block',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 3 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Mini Cart → Can see the icon with a badge', () => {
+	it( 'Shopper can see the Mini Cart icon and it badge on the front end', async () => {
+		await page.goto( await getBlockPagePermalink( block.name ), {
+			waitUntil: 'networkidle0',
+		} );
```

Comment:
This is quite slow because we need to:
1. Search for the Mini Cart block page in the page list table.
2. Go to the edit page of Mini Cart page.
3. Open the sidebar if needed.
4. Toggle the Permalink panel.
5. Get the link of the Mini Cart page.
6. Navigate to the page using `page.goto()`.

The bellow suggestion is an alternative way to navigate to the Mini Cart Block page.
Given that all the pages are added to the Storefront header navigation, we can navigate to the block page by clicking on the corresponding nav menu item. This sounds less reliable than the current approach but it's fine as long as we don't remove the menu items from the navigation.
```suggestion
		await page.goto( createURL( '/' ), { waitUntil: 'networkidle0' } );
		await expect( page ).toClick( '.nav-menu a', { text: block.name } );
		await page.waitForNavigation();
```